.gitignore

```diff
@@ -43,7 +43,7 @@ package-lock.json
 #build and dist files
 /build/
 /dist/
-/cnx-recipes.egg-info/
+/ce-styles.egg-info/
```

Comment:
@kswanson33 I have checked that this is a python related file. I'm not sure if we will need it for something, just wanted to let you know.